www/src/components/masthead.js

```diff
@@ -17,9 +17,12 @@ const MastheadContent = () => (
   <div
     className="masthead-content"
     css={{
-      margin: `0 ${space[8]}`,
+      boxSizing: `border-box`,
```

Comment:
We should have this set already via Typography.js's https://github.com/KyleAMathews/typography.js/blob/1a3bfc103d41d59f04e89573c2c87e1d95abdd12/packages/typography/src/utils/createStyles.js#L63-L66